server/api/users/routes/get_users.js

```diff
@@ -30,7 +31,7 @@ module.exports = (server) => ({
       page: page || 0,
       include_totals: true,
       fields: 'user_id,name,email,identities,picture,last_login,logins_count,multifactor,blocked',
-      search_engine: 'v2'
+      search_engine: (config('AUTH0_RTA').replace('https://', '') !== 'auth0.auth0.com') ? 'v2' : 'v3'
```

Comment:
I think you can use `window.config.SEARCH_ENGINE` here

Reply:
@siacomuzzi unless I'm missing something, this on the server side and doesn't have access to the `window` variable?

Reply:
I was talking about the `config` object, but I think `SEARCH_ENGINE` is not available there